03-task-manager/optionalArrayMethodsReviewExtraAssignment.js

```diff
@@ -147,13 +149,58 @@ object.behavior()
 // changes to this file with your MR for week 3.
 
 const names = [
-  'Dimitry SantiAgo',
-  'Carlos d. Perez',
-  'tam  person',
-  'Mariana Gomez',
-  'Amy You'
+  "Dimitry SantiAgo",
+  "Carlos d. Perez",
+  "tam  person",
+  "Mariana Gomez",
+  "Amy You",
+  "Timothy      Cook",
+  "Nick_Masters",
+  "Timmy-Turner",
+  "Billy\nBob",
 ];
 
+// Create a new array with only each person's last name
+// Filter names that don't match the format "<first> <last>"
+
+const validNames = names.filter((name) => name.split(" ").length === 2);
+
+//splitting each name by the space character and returning the last element of the resulting array
+const familyNames = validNames.map((familyName) => familyName.split(" ")[1]);
+
+console.log("Create a new array with only each person's last name",familyNames);
+
+
+// - Create a new array where everyone's name is converted to "Title Case"
+//   - The first character of each word should be uppercase
+//   - All other characters in the word should be lowercase
+//   - expected output is ['Dimitry Santiago', 'Carlos D. Perez', 'Tam Person', ...]
+
+const formattedNames = names.map((name) => name.replace(/\s+/g, " ").trim());
+const titleCaseNames = formattedNames.map((name) =>
```

Comment:
Nice! If you wanted to use this in a larger program, this is a great candidate to pull out as a separate function (not expected or needed for this assignment)